src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -1658,7 +1672,7 @@ public void SarifTransformerTests_ToCurrent_CodeFlows()
               ""message"": ""Path from declaration to usage"",
               ""locations"": [
                 {
-                  ""kind"": ""declaration"",
+                  ""kind"": ""call"",
```

Comment:
Why these changes? The `snippet` shows that this line is not in fact a function call.

Reply:
I wanted the v1->v2 log to match the v2->v1 log, as far as the kind vs. nestingLevel is concerned

Reply:
But these are just wrong (the `kind` is inconsistent with the `snippet`).

Reply:
Yeah I get that, I'll adjust.

Reply:
In fact, if you have a v2 `threadFlowLocation` whose `indentLevel` has increased by 1, then the _previous_ `threadFlowLocation` must have been the call site.